### PR TITLE
ODF UBI image update

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

**Review by akgunjal — APPROVED** [review]
LGTM

File: .secrets.baseline [hunk 1]
```diff
@@ -414,7 +477,7 @@
       }
     ]
   },
-  "version": "0.13.1+ibm.51.dss",
+  "version": "0.13.1+ibm.47.dss",
```
Comment (mssachan) [inline .secrets.baseline]: @gayathrimenath-ibm Please upgrade `detect-secrets` tool and regenerate this file
`pre-commit autoupdate`
https://github.ibm.com/Whitewater/whitewater-detect-secrets/wiki/Developer-Tool-FAQs#how-do-i-upgrade-detect-secrets-to-a-newer-version

File: config-templates/redhat/odf-local/changelog.json [hunk 2]
```diff
@@ -3,6 +3,15 @@
         "4.11": {
             "status": "supported",
             "latest-revision": {
+                "revision": "2",
+                "release-date": "20 Feburary 2023",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: `Feburary` --> `February`

File: config-templates/redhat/odf-local/changelog.json [hunk 3]
```diff
@@ -3,6 +3,15 @@
...
+                "cves": "CVE-2022-47629",
+                "golang": "",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: Add current golang version of the image

File: config-templates/redhat/odf-local/changelog.json [hunk 4]
```diff
@@ -17,20 +26,20 @@
...
             "latest-revision": {
-                "revision": "12",
-                "release-date": "23 January 2023",
-                "cves": "CVE-2022-43680, CVE-2022-42010, CVE-2022-42011, CVE-2022-42012, CVE-2022-3821, CVE-2022-35737, CVE-2021-46848",
-                "golang": "1.18.9",
-                "ubi": "8.7-1049",
+                "revision": "13",
+                "release-date": "20 Feburary 2023",
+                "cves": "CVE-2022-47629",
+                "golang": "",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: Same here

File: config-templates/redhat/odf-local/changelog.json [hunk 5]
```diff
@@ -17,20 +26,20 @@
         "4.10": {
             "status": "supported",
             "latest-revision": {
-                "revision": "12",
-                "release-date": "23 January 2023",
-                "cves": "CVE-2022-43680, CVE-2022-42010, CVE-2022-42011, CVE-2022-42012, CVE-2022-3821, CVE-2022-35737, CVE-2021-46848",
-                "golang": "1.18.9",
-                "ubi": "8.7-1049",
+                "revision": "13",
+                "release-date": "20 Feburary 2023",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: same typo here

File: config-templates/redhat/odf-remote/changelog.json [hunk 6]
```diff
@@ -3,6 +3,15 @@
         "4.11": {
             "status": "supported",
             "latest-revision": {
+                "revision": "2",
+                "release-date": "20 Feburary 2023",
+                "cves": "CVE-2022-47629",
+                "golang": "",
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: Same here as mentioned above

**Review by mssachan — CHANGES_REQUESTED** [review]

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**